storage/remote/client_test.go

```
@@ -64,7 +64,7 @@ func TestStoreHTTPErrorHandling(t *testing.T) {
 			panic(err)
 		}
 
-		c, err := NewClient(0, &clientConfig{
+		c, _ := NewClient(0, &clientConfig{
```

Let's not ignore the error.